From 4393c6a48a4d9a17a0d9e21c0fb933df7773041a Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Thu, 3 Feb 2022 17:03:22 +0200
Subject: [PATCH 54/64] configs: Remove CONFIG_SYS_ERRATUM_ERR050543 from RDB3

Removed the CONFIG_SYS_ERRATUM_ERR050543 from RDB3 boards'
defconfigs, since the erratum has been solved in HW for
S32G3-based platforms.

Issue: ALB-8353
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 configs/s32g399ardb3_defconfig      | 1 -
 configs/s32g399ardb3_qspi_defconfig | 1 -
 2 files changed, 2 deletions(-)

diff --git a/configs/s32g399ardb3_defconfig b/configs/s32g399ardb3_defconfig
index 188e31b454..43c32857aa 100644
--- a/configs/s32g399ardb3_defconfig
+++ b/configs/s32g399ardb3_defconfig
@@ -4,7 +4,6 @@ CONFIG_ENV_SIZE=0x2000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G399ARDB3=y
-CONFIG_SYS_ERRATUM_ERR050543=y
 CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
diff --git a/configs/s32g399ardb3_qspi_defconfig b/configs/s32g399ardb3_qspi_defconfig
index 6214c46f5d..4f577c93cb 100644
--- a/configs/s32g399ardb3_qspi_defconfig
+++ b/configs/s32g399ardb3_qspi_defconfig
@@ -5,7 +5,6 @@ CONFIG_ENV_SECT_SIZE=0x10000
 CONFIG_ENV_OFFSET=0x1e0000
 CONFIG_NR_DRAM_BANKS=3
 CONFIG_TARGET_S32G399ARDB3=y
-CONFIG_SYS_ERRATUM_ERR050543=y
 CONFIG_S32_ATF_BOOT_FLOW=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_TOOLS_DEBUG=y
-- 
2.17.1

